styleguide/_themes/derek/scss/components/mainnav.scss

```diff
@@ -0,0 +1,371 @@
+//main nav
+.main-nav {
+  background: $dark-gray;
+  border: 0;
+  border-radius: 0;
+  margin: 0;
+
+  .navbar-menuContainer {
```

Comment:
Just making a note here for myself and other code reviewers that this nesting was deemed necessary due to bootstrap stuff, and do in fact pass the sass-lint process, so there is no need to relocate it after all.